cylc/flow/cfgspec/globalcfg.py

```diff
@@ -31,13 +32,23 @@
 )
 from cylc.flow.parsec.exceptions import ParsecError
 from cylc.flow.parsec.upgrade import upgrader
+from cylc.flow.parsec.util import printcfg
 from cylc.flow.parsec.validate import (
     CylcConfigValidator as VDR,
     DurationFloat,
     Range,
     cylc_config_validate,
 )
 
+
+PLATFORM_REGEX_TEXT = '''
+Names from the configuration are regular expressions.
+Any match is a valid platform.
+Cylc searches the definitions from the bottom upwards.
+If a platform name matches a regex on the list Cylc will
+stop searching.'''
```

Comment:
Fine, but could be more concise.

```suggestion
PLATFORM_REGEX_TEXT = '''
Configured names are regular expressions; any match is a valid platform.
The are searched from the bottom up, until the first match is found.'''
```